inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -115,16 +115,91 @@ void LayerTestsCommon::LoadNetwork() {
     executableNetwork = core->LoadNetwork(cnnNetwork, targetDevice);
 }
 
+template<InferenceEngine::Precision::ePrecision PRC>
```

Comment:
remove printBuffer related codes, you shall use it personally, then we can focus on the key codes